### ISSUE TITLE
missing library error on MacOS

### ISSUE THREAD

**Issue by Kelvinrr** [USGS-Astrogeology/swigcsm#9]
```
Traceback (most recent call last):
  File "/Users/krodriguez-pr/anaconda3/envs/bayleef/lib/python3.6/site-packages/csmapi/csmapi.py", line 14, in swig_import_helper
    return importlib.import_module(mname)
  File "/Users/krodriguez-pr/anaconda3/envs/bayleef/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 658, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 571, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 922, in create_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ImportError: dlopen(/Users/krodriguez-pr/anaconda3/envs/bayleef/lib/python3.6/site-packages/csmapi/_csmapi.so, 2): Library not loaded: @rpath/libcsmapi.dylib.3.0.3
  Referenced from: /Users/krodriguez-pr/anaconda3/envs/bayleef/lib/python3.6/site-packages/csmapi/_csmapi.so
  Reason: image not found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 3, in <module>
    import autocnet
  File "/Users/krodriguez-pr/repos/autocnet/autocnet/__init__.py", line 11, in <module>
    import autocnet.graph
  File "/Users/krodriguez-pr/repos/autocnet/autocnet/graph/__init__.py", line 1, in <module>
    from . import edge
  File "/Users/krodriguez-pr/repos/autocnet/autocnet/graph/edge.py", line 12, in <module>
    from autocnet.graph.node import Node
  File "/Users/krodriguez-pr/repos/autocnet/autocnet/graph/node.py", line 6, in <module>
    from csmapi import csmapi
  File "/Users/krodriguez-pr/anaconda3/envs/bayleef/lib/python3.6/site-packages/csmapi/__init__.py", line 1, in <module>
    from csmapi.csmapi import *
  File "/Users/krodriguez-pr/anaconda3/envs/bayleef/lib/python3.6/site-packages/csmapi/csmapi.py", line 17, in <module>
    _csmapi = swig_import_helper()
  File "/Users/krodriguez-pr/anaconda3/envs/bayleef/lib/python3.6/site-packages/csmapi/csmapi.py", line 16, in swig_import_helper
    return importlib.import_module('_csmapi')
  File "/Users/krodriguez-pr/anaconda3/envs/bayleef/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named '_csmapi'
(bayleef) igswzawglt-krod:autocnet krodriguez-pr$ conda list csm
# packages in environment at /Users/krodriguez-pr/anaconda3/envs/bayleef:
#
# Name                    Version                   Build  Channel
csmapi                    0.1.0                    py36_0    usgs-astrogeology
libcsm                    3.0.3                h04f5b5a_2    usgs-astrogeology
```

Quick fix for now is to copy libcsmapi.dylib.3 libcsmapi.dylib.3.0.3 
Seems the problem is a simple symlink missing during the conda build process.

**Comment by jlaura** [comment]
This is a fresh install off of anaconda?  Just so I know where the break is.

**Comment by Kelvinrr** [comment]
yes, Adam and I run into it after doing a fresh anaconda install and attempting to use Autocnet (on import).

**Comment by jlaura** [comment]
This is going to wait until a pull from the conda-forge release of the csm library. https://github.com/conda-forge/staged-recipes/pull/7528

**Comment by jlaura** [comment]
should be closed by #13
